tests/unit/core/components/TestPaginate.js

```diff
@@ -191,54 +176,11 @@ describe('<Paginate />', () => {
       queryParams,
     });
 
-    const links = scryRenderedComponentsWithType(root, PaginatorLink);
+    const firstLink = root.find(PaginatorLink).first();
     // Just do a quick sanity check on the first link.
-    expect(links[0].props.LinkComponent).toEqual(LinkComponent);
-    expect(links[0].props.queryParams).toEqual(queryParams);
-    expect(links[0].props.currentPage).toEqual(currentPage);
-    expect(links[0].props.pathname).toEqual(pathname);
-    expect(links[0].props.pageCount).toEqual(pageCount);
-  });
-
-  it('renders the right links', () => {
-    const pathname = '/some-path/';
-
-    class PaginateWrapper extends React.Component {
-      render() {
-        const props = {
-          ...getRenderProps(),
-          count: 250,
-          currentPage: 5,
-          showPages: 5,
-          pathname,
-        };
-        return <Paginate {...props} />;
-      }
-    }
-
-    function renderPaginateRoute() {
-      return new Promise((resolve) => {
-        const node = document.createElement('div');
-        render((
-          <Router history={createMemoryHistory('/')}>
-            <Route path="/" component={PaginateWrapper} />
-          </Router>
-        ), node, () => {
-          resolve(node);
-        });
-      });
-    }
-
-    return renderPaginateRoute().then((root) => {
-      const links = Array.from(root.querySelectorAll('a'));
-      expect(links.map((link) => [link.textContent, link.getAttribute('href')])).toEqual([
```

Comment:
Did this get moved somewhere? I know it wasn't a great test, just making sure you deleted it on purpose.

Reply:
I checked and those code paths are covered in the `PaginateLink` tests and in other tests in this file. They're just leftovers from a much earlier time and they aren't needed anymore.